src/components/table/README.md

```diff
@@ -901,6 +902,52 @@ event will not be emitted when the input, select, textarea is clicked (unless th
 (even when disabled)
 
 
+## Row select
+You can make rows selectable, by using the prop `selectable`.
+
+By default, any row clicked is toggled, any other deselected. the SHIFT key selects a range of
+rows, and CTRL/CMD click will toggle the selected row.
+
+The prop `select-mode` can change the selecting mode:
+ - `multi`: each click will select/deselect the row
+ - `single`: only a single row can be selected at a time
+
+When a table is `selectable` and the user clicks on a row, `<b-table>` will emit the `row-selected`
+event, passing a single argument which is the complete list of selected items.
+**Treat this argument as read-only.**
+
+**Note:** _Paging or sorting will clear the selection._
+
+```html
+<template>
+  <b-table selectable select-mode="os" :items="items" @row-selected="rowSelected"></b-table>
```

Comment:
Ans what is `select-mode="os"`?  Shouldn't it be one of `single` or `multi`?

Reply:
Yeah, as the mode you are using is not documented as to how it works... and if it is a "default" fallback, then it probably should have an explicit mode name, and any other mode that isn't supported falls back to it.

Reply:
I though this text would describe it, but it seems it is not clear enough, I'll see how I can rephrase it:

> By default, any row clicked is toggled, any other deselected. the SHIFT key selects a range of rows, and CTRL/CMD click will toggle the selected row.

Reply:
Maybe make a default mode of `range`... although I think `single` mode should be the default.

Reply:
But the default mode needs a name.

Reply:
I rephrased that part, is it better (not yet changed the default mode though)?

Reply:
And maybe bind the mode to a select so users can change the mode to see the differences in the example.

We also need to mention that selection only works for the displayed rows, and is cleared when pagination or filtering happens.